yt_dlp/extractor/nbc.py

```diff
@@ -379,14 +380,16 @@ class NBCNewsIE(ThePlatformIE):  # XXX: Do not subclass from concrete IE
     _TESTS = [
         {
             'url': 'http://www.nbcnews.com/watch/nbcnews-com/how-twitter-reacted-to-the-snowden-interview-269389891880',
-            'md5': 'cf4bc9e6ce0130f00f545d80ecedd4bf',
+            'md5': 'fb3dcd2d7b1dd9804305fa2fc95ab610',  # note: for some reason, downloading this video with the --test command doesn't give the same md5. Seems to be the right video though.
```

Comment:
are you updating these tests by running from master or your patch in #7484?

Reply:
I've left patch #7484 separate from this, and I've made it so all tests run from master. Idea being: make sure everything works in master. If something doesn't work anymore after I've applied #7484, then I know it broke something. I also intend to keep them as separate branches so they can be discussed independantly.